core/core.go

```diff
@@ -398,6 +402,7 @@ func (t *TextileNode) Start() error {
 	} else {
 		log.Info("desktop node is ready")
 	}
+	t.online = true
```

Comment:
using this flag to determine "online-ness" vs. just whether or not the ipfs node is nil, since we need to set it as non-nil in start _before_ blocking. so, instead we can wait till the very end to mark the textile node as online

Reply:
ah, sweet. this might be what i was asking